docs/tutorial/notebook/reading_a_config_file.ipynb

```diff
@@ -0,0 +1,608 @@
+{
```

Comment:
<p>Checking the type is not useful. Instead use</p><p><code>vars(cfg) </code></p><p>This will list the 3 dictionaries in cfg</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done.</p>